packages/guui/package.json

```diff
@@ -4,27 +4,43 @@
   "description": "Guardian rendering framework",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "prepublish": "cd ../.. && make pre-publish-guui",
+    "postpublish": "cd ../.. && make post-publish-guui"
+  },
+  "files": [
+    "document.js",
+    "grid.js",
+    "htmlTemplate.js",
+    "index.js"
+  ],
+  "directories": {
+    "lib": "./lib",
+    "components": "./components"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@guardian/guui": "2.0.0-alpha.1",
```

Comment:
What was the thinking behind deleting this line?

It always seemed to me a strange concept, specifying a package as a dependency of itself, although I appreciate this saves us having to hardcode `react` and `react-emotion` as module specifiers in all Guui components.

Reply:
It also seemed strange to me that the `guui` library uses `react` and `react-emotion` via a copy of itself imported as dependency. Let's say we wanted to upgrade `React` in `guui` to an api breaking version which requires updates to all our components, how would we do that if the version of `React` being used is being pulled from the dependency? You'd want to update the components in the same version of `guui` that you update `React` surely, making this change makes that possible.

The overhead of having to import `react` and `react-emotion` from the actual `node_modules` seems negligible, and I like the idea of introducing a `make create component` command which basically sets up a new component (with co-located tests/README.md etc.) which'd take away the overhead of setting up a new component.